addons/udes_get_info/models/models.py

```diff
@@ -40,7 +40,7 @@ def get_info(self, level=1, info_fields=frozenset(), extra_fields=frozenset(), *
 
 
 @add_if_not_exists(models.BaseModel)
-def _get_info(self, level, info_fields, extra_fields):
+def _get_info(self, level, info_fields=frozenset(), extra_fields=frozenset(), **kwargs):
```

Comment:
Why do we need this change?  We shouldn't be calling `_get_info` directly, and `get_info` always passes these arguments.